### DISCUSSION TITLE
Installing Webots on AWS EC2 instances

### DISCUSSION THREAD

**Discussion by tuncalie** [cyberbotics/webots#4816] [General]
I am trying to install Webots on an AL2_x86_64 EC2 instance and couldn't get dependencies together for compiling from code as I assume it is the only way to install in an AL2 instance. I would appreciate pointers on how to do the installation.

**Comment by ygoumaz** [comment]
Hi! Are you accessing your EC2 instance via SSH? Are you trying to install and run a single instance of Webots on your EC2 machine or is your goal to run multiple instances in parallel? Because there are more suitable AWS services for this kind of projects.

To install Webots on your EC2 instance, the best way is to access it via SSH and follow the instructions on the [Linux installation wiki page](https://github.com/cyberbotics/webots/wiki/Linux-installation/). The process will ensure that all necessary dependencies are installed. I have never tried to install it on AL2 but an Ubuntu 20.04 instance will work for sure.
